src/manual/devel/namespaces.md

```diff
@@ -188,6 +188,7 @@ Worker types are specific to the provisioner ID, but provisioners that provide g
 * `tutorial` - default worker type for the getting-started tutorial in this documentation
 * `github-worker` - default worker type for Github-triggered jobs
 * `hg-worker` - default worker type for Mercurial-triggered jobs
+* `buildbot-bridge` - worker type for tasks to be executed using the Buildbot Bridge
```

Comment:
I think that comes down to taskcluster/taskcluster-rfcs#22 - we should have something that enumerates all worker types.